src/apps/workspaces/crud/user_applet_access.py

```diff
@@ -931,6 +931,7 @@ async def pin(
         pin_role: UserPinRole,
         pinned_user_id: uuid.UUID | None,
         pinned_subject_id: uuid.UUID | None,
+        force_unpin: bool = False,
```

Comment:
I'm confused by the use of this arg.
Previously this function would either create or delete a pin if a pin already existed, sort of like `toggle_pin`, but now you're using `pin(..., force_unpin=True)` to essentially unpin a user.

Would it make sense to restore the deleted lines 958-964 below for `unpin()` and make the function's purpose explicit?

Reply:
I remove unpin() function because it not used anywhere and it not actually delete the row in user_pins table. 

I added force_unpin to pin() function to make sure it will not create a new record, because I want a unpin participant if exists, but not create a new.

I almost all code for unpin() method will be the same, so I choose to not duplicate this code.

Reply:
I get the decision to avoid duplicating code, but it still feels wrong to me to use `pin()` to remove a pin, but I can live with it 😆